numpy/core/_asarray.py

```diff
@@ -0,0 +1,323 @@
+"""
+Functions in the ``as*array`` family that promote array-likes into arrays.
+
+`require` fits this category despite its name not matching this pattern.
+"""
+from __future__ import division, absolute_import, print_function
+
+from .overrides import set_module
+from .multiarray import array
+
+
+__all__ = [
```

Comment:
I still prefer normal lists fromatting for this. This sort of works for a few entries, but I don't think it buys much over the usual left to right reading habit.

Reply:
> This sort of works for a few entries

That's a feature, it discourages too many items from being put in a single module ;)